rest-api-spec/src/main/resources/rest-api-spec/api/delete_script.json

```diff
@@ -28,7 +28,15 @@
       },
       "master_timeout":{
         "type":"time",
-        "description":"Specify timeout for connection to master"
+        "description":"Specify timeout for connection to master",
+        "deprecated":{
+          "version":"2.0.0",
+          "description":"To promote inclusive language, use 'cluster_manager_timeout' instead."
```

Comment:
It feels to me promote is the not the right work. may be to support inclusive language. or to be more inclusive, use ..

Reply:
Thanks for your opinion! I will change the word `promote` to `support`.

Reply:
Changed in commit https://github.com/opensearch-project/OpenSearch/pull/2682/commits/7d0de096a0b1213ae9ba39bfa88623b699e92973